Core/Core/Profile/ProfileViewController.swift

```diff
@@ -104,7 +104,9 @@ public class ProfileViewController: UIViewController, ProfileViewProtocol {
         }
 
         env.api.makeRequest(GetUserRequest(userID: "self")) { [weak self] user, _, _ in
-            self?.avatarButton?.isHidden = user?.permissions?.can_update_avatar == false
+            performUIUpdate {
+                self?.avatarButton?.isHidden = user?.permissions?.can_update_avatar == false
+            }
```

Comment:
This fixes an error I was seeing every time I opened the profile drawer.

Reply:
good catch